src/main/java/org/generationcp/middleware/dao/dms/DmsProjectDao.java

```diff
@@ -1470,6 +1499,10 @@ public long countStudies(final StudySearchFilter studySearchFilter) {
 			studyInstanceDto
 				.setActive(((Integer) result.get("active")) == 1 ? Boolean.TRUE.toString() : Boolean.FALSE.toString());
 			studyInstanceDtoList.add(studyInstanceDto);
+
+			final Map<String, String> properties = new HashMap<>();
+			properties.put("studyObjective", result.get("studyObjective") ==  null ? "" : String.valueOf(result.get("studyObjective")));
+			studyInstanceDto.setAdditionalInfo(properties);
```

Comment:
Can be simplified as:
`studyInstanceDto.setAdditionalInfo(Collections.singletonMap("studyObjective", result.get("studyObjective") ==  null ? "" : String.valueOf(result.get("studyObjective")));`

Reply:
Move `setAdditionalInfo` before `studyInstanceDto` is added to list

Reply:
> Can be simplified as:
> `studyInstanceDto.setAdditionalInfo(Collections.singletonMap("studyObjective", result.get("studyObjective") == null ? "" : String.valueOf(result.get("studyObjective"))));`

Hi @darla-leafnode if a singletonMap is used, adding more values to the map produces an exception.